Assets/BossRoom/Scripts/Server/Game/Character/ServerCharacterMovement.cs

```diff
@@ -125,6 +125,34 @@ public void CancelMove()
             m_MovementState = MovementState.Idle;
         }
 
+        /// <summary>
+        /// Instantly moves the character to a new position. NOTE: this cancels any active movement operation!
+        /// Typically this includes an RPC that will let clients know that this movement happened due to a teleport,
+        /// so that they don't try to animate the character traveling to the new location. But if the teleport occurs
+        /// as part of an Action that already handles the visualization (such as in DashAttackActionFX), this extra
+        /// RPC isn't necessary and can be omitted.
+        /// </summary>
+        /// <param name="newPosition">new coordinates the character should be at</param>
+        /// <param name="sendClientRpc">whether we should notify clients with an RPC</param>
+        public void Teleport(Vector3 newPosition, bool sendClientRpc)
```

Comment:
from what I can see, sendClientRpc is always false. YAGNI would apply here

Reply:
ok